modules/installation-rhv-overriding-some-default-behaviors.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installation-rhv-overriding-some-default-behaviors_{context}"]
+= Overriding some default behaviors
+
+Here, you use three python scripts to override some of the installation program's default behaviors:
+
+* By default, the installation program uses the machine API to create nodes. To override this default behavior, you set the number of compute nodes to zero replicas. Later, you use Ansible playbooks to create the compute nodes.
```

Comment:
@santos1709  This is confusing. We have choose one way to do this and then tell users what to do. @jan-zmeskal Can you comment?

Reply:
@santos1709 I think @rolfedh is right. If I remember correctly, this is about _compute_ nodes, not _master_ nodes. In IPI context, you're of course right, number of compute nodes shouldn't be 0. But here we do it by design. But maybe we're just misunderstanding your comment.

Reply:
I think you guys misunderstand the comment, it's not anything serious, but I meant that in the same document on section "Making control-plane nodes unschedulable" (line 8 of modules/installation-rhv-making-control-plane-nodes-unschedulable.adoc) it says "Avoid setting compute `replicas` to zero because this makes control-plane nodes schedulable. Instead, to make the control-plane nodes unschedulable, modify `manifests/cluster-scheduler-02-config.yml` by setting `masterSchedulable` to `False`"
And given that the "Making control-plane nodes unschedulable" comes after this part we are commenting about, I just thought the some mention to the "Making control-plane nodes unschedulable" should be given here to create any confusion